hasher-matcher-actioner/src/OpenMediaMatch/templates/components/banks/banks_grid.html.j2

```diff
@@ -0,0 +1,121 @@
+{# Copyright (c) Meta Platforms, Inc. and affiliates. #}
+<h1>Banks</h1>
+<p class="text-secondary">Banks are like folders. You can do the following on this page:</p>
+<ul class="text-secondary">
+    <li>Create new banks</li>
+    <li>Add content: Add content to each bank</li>
+    <li>Search by content: Find banks that contain a certain piece of content</li>
+</ul>
```

Comment:
I made this up lol. Someone should come up with better copy to describe these concepts.

I think all the pages should have a h1 title and a description